examples/rllib_example.py

```diff
@@ -0,0 +1,116 @@
+# Rllib Example for single and multi-agent training for GodotRL with onnx export,
+# needs rllib_config.yaml in the same folder or --config_file argument specified to work.
+
+import argparse
+import os
+import pathlib
+
+import ray
+import yaml
+from ray import train, tune
+from ray.rllib.algorithms.algorithm import Algorithm
+from ray.rllib.env.wrappers.pettingzoo_env import ParallelPettingZooEnv
+from ray.rllib.policy.policy import PolicySpec
+
+from godot_rl.core.godot_env import GodotEnv
+from godot_rl.wrappers.petting_zoo_wrapper import GDRLPettingZooEnv
+from godot_rl.wrappers.ray_wrapper import RayVectorGodotEnv
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(allow_abbrev=False)
+    parser.add_argument("--config_file", default="rllib_config.yaml", type=str, help="The yaml config file")
```

Comment:
I think this should be `examples/rllib_config.ymal`

Reply:
I usually call the example from within the examples folder, so the default was based on my usage. If calling from GDRL repository directly then it should be changed. 

If someone installs GDRL using `pip install` and then just downloads the example file and config file, they might not have the entire repository, but I'm not sure how common this is.

I leave this up to you, I can definitely change the default.